### PR TITLE
Various Spelling Corrections

### PR DESCRIPTION
**Pull request by CraigCraig**
Various Spelling Corrections

### CODE REVIEW

**Comment by CraigCraig** [comment]
I'm not sure if my other commits are supposed to show like that. If there's something I'm doing wrong, please let me know. I'm pretty new with Git.

File: src/util/util.c [hunk 1]
```diff
@@ -41,7 +41,7 @@ int metres_to_feet(int metres)
 
 int mph_to_kmph(int mph)
 {
-	// 1 mph = 1.60934 kmph
+	// 1 mph = 1.60934 kph
```
Comment (Lastorder-DC) [inline src/util/util.c]: kmph is right(km/h)

Comment (CraigCraig) [inline src/util/util.c]: I guess both are correct, however I went with the more common one.
I suppose this is up to personal opinion, though it's just a comment line, so it doesn't really matter anyways I suppose.
From now on I'll accept both forms when spellchecking.
Thank you.

File: distribution/changelog.txt [hunk 2]
```diff
@@ -187,7 +187,7 @@
 - Feature: Finance window accessible from toolbar (enabled in settings).
 - Feature: Research and development / research funding now accessible as a stand alone window without the requirement of the finances window.
 - Feature: Extra viewport windows.
-- Feature: Park window viewport is resizable.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: `resizeable` isn't wrong per se, but not preferential spelling.

File: src/util/util.c (hunk 1 above)
Comment (Gymnasiast) [inline src/util/util.c]: kph is colloquial, but not correct. kmph is correct but not very pretty. km/h is the most common (except for the UK) and also correct.

**Comment by Gymnasiast** [comment]
I left some remarks. If you have trouble opening a clean PR, we can help you on Gitter. Or I can merge your corrections manually.

**Comment by CraigCraig** [comment]
Thank you! If I continue to have an issue I'll join up with you guys on that chat.

**Pull request closed without merging**